### PR TITLE
Add note about rounding to non-text contrast understanding, fix `Success Criteria` to `Success Criterion`

### PR DESCRIPTION
**Pull request by patrickhlauke**
In discussion for https://github.com/w3c/wcag/issues/2251 it emerged that changes were pushed in 2016 to the understanding documents for 1.4.3 and 1.4.6, clarifying that values should not be rounded when evaluating pass/fail for contrast ratios - see https://github.com/w3c/wcag/issues/2251#issuecomment-1061917901

However, it now appears that this change at the time somehow fell through the cracks in the various reorganisations of the repositories here. And indeed, the two notes that were agreed by AGWG and merged at the time are nowhere to be found - not in any of these

* https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html
* https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast7.html
* https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum
*  https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced
* https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum
* https://www.w3.org/WAI/WCAG22/Understanding/contrast-enhanced

This PR reintroduces those notes (since they were already agreed back in 2016, and are clearly still valuable) to 1.4.3 and 1.4.6, and also adds a matching note to 1.4.11.

Ideally, this note should just be added once to the contrast ratio definition https://w3c.github.io/wcag/21/guidelines/#dfn-contrast-ratio ... but that would be a change of normative wording which is doubtful will get through. so, while duplicative, this at least adds it to the non-normative understanding documents.

This note should not only go into 2.2, but also 2.1 (and ideally 2.0, but i doubt these get touched anymore)

EDIT: after merging main back into this PR, and some parallel changes that got pushed there earlier, this PR is now just correcting `Success Criteria` to `Success Criterion` (singular) in the contrast-minimum and contrast-enhanced understanding, and then adding the same note to non-text contrast understanding

### CODE REVIEW

**Review by michael-n-cooper — APPROVED** [review]
I merged main into this because it crossed in the mail with 6a67a5c90c7ee985ea56d8d875e6570143e1b32d, as far as I can tell otherwise ok

File: understanding/20/contrast-enhanced.html [hunk 1]
```diff
@@ -67,7 +67,10 @@ <h2>Intent of Contrast (Enhanced)</h2>
             resulting image is roughly equivalent to 1.2 and 1.5 em or to 120% or 150% of the
             default size for body text as rendered by the browser.
          </p>
-      	<p>The 7:1 and 4.5:1 contrast ratios referenced in this Success Criteria are intended to be treated as threshold values. When comparing the computed contrast ratio to the Success Criteria ratio the computed values should not be rounded (eg. 4.499:1 would not meet the 4.5:1 threshold).</p>
+         <p>The 7:1 and 4.5:1 contrast ratios referenced in this Success Criterion are intended to be
+            treated as threshold values. When comparing the computed contrast ratio to the Success Criterion
+            ratio, the computed values should not be rounded (e.g. 4.499:1 would not meet the 4.5:1 threshold).</p>
```
Comment (patrickhlauke) [inline understanding/20/contrast-enhanced.html:72]: it seems that after merging the main branch back into this (https://github.com/w3c/wcag/pull/2254#pullrequestreview-923310979), the only real change here and in the contrast minimum file is that it's split over multiple lines, and changes the old `Success Criteria` to `Success Criterion` (singular)

File: understanding/20/contrast-minimum.html [hunk 2]
```diff
@@ -61,7 +61,10 @@ <h2>Intent of Contrast (Minimum)</h2>
             to use approximately 19 pt and 24 pt font sizes in order to successfully present images
             of large-scale text to a user. 
          </p>
-      	<p>The 4.5:1 and 3:1 contrast ratios referenced in this Success Criteria are intended to be treated as threshold values. When comparing the computed contrast ratio to the Success Criteria ratio the computed values should not be rounded (eg. 4.499:1 would not meet the 4.5:1 threshold).</p>
+         <p>The 3:1 and 4.5:1 contrast ratios referenced in this Success Criterion are intended to be
+            treated as threshold values. When comparing the computed contrast ratio to the Success Criterion
```
Comment (patrickhlauke) [inline understanding/20/contrast-minimum.html:65]: see above comment

File: understanding/21/non-text-contrast.html [hunk 3]
```diff
@@ -15,6 +15,10 @@ <h2>Intent</h2>
 
 			<p>Low contrast controls are more difficult to perceive, and may be completely missed by people with a visual impairment. Similarly, if a graphic is needed to understand the content or functionality of the webpage then it should be perceivable by people with low vision or other impairments without the need for contrast-enhancing assistive technology.</p>
 
+			<div class="note">
+				<p>The 3:1 contrast ratios referenced in this Success Criterion is intended to be treated as threshold values. When comparing the computed contrast ratio to the Success Criterion ratio, the computed values should not be rounded (e.g. 2.999:1 would not meet the 3:1 threshold).</p>
+			</div>
+
```
Comment (patrickhlauke) [inline understanding/21/non-text-contrast.html:21]: this change, however, is new and for should be merged in

**Comment by patrickhlauke** [comment]
so, is this good to merge?

**Comment by alastc** [comment]
@michael-n-cooper can you make sure this is applied to 2.1 docs as well please? (Does deleting the branch make that harder?)

**Comment by michael-n-cooper** [comment]
I just update from main, all set.

**Pull request merged**